### PR TITLE
OpenCL support

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: dockcross-manylinux-build-module-wheels-opencl.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  rm -rf ITKPythonPackage/standalone-x64-build/ITKs/Modules/Core/GPUCommon/
+fi
+
+# Build wheels
+mkdir -p dist
+DOCKER_ARGS="-e ELASTIX_USE_OPENCL=1 -v $(pwd)/dist:/work/dist/ -v $(pwd)/ITKPythonPackage:/ITKPythonPackage -v $(pwd)/tools:/tools -v $(pwd)/OpenCL-ICD-Loader/inc/CL:/usr/include/CL -v $(pwd)/OpenCL-ICD-Loader-build/libOpenCL.so.1.2:/usr/lib64/libOpenCL.so.1 -v $(pwd)/OpenCL-ICD-Loader-build/libOpenCL.so.1.2:/usr/lib64/libOpenCL.so"
+/tmp/dockcross-manylinux-x64 \
+  -a "$DOCKER_ARGS" \
+  "/ITKPythonPackage/scripts/internal/manylinux-build-module-wheels.sh" "$@"
+#/tmp/dockcross-manylinux-x86 \
```
Comment (dzenanz) [inline dockcross-manylinux-build-module-wheels-opencl.sh]: The commented code can be removed?

File: test/itkElastixRegistrationMethodTest.cxx [hunk 2]
```diff
@@ -77,16 +77,26 @@ int itkElastixRegistrationMethodTest( int argc, char * argv[] )
...
   ReaderType::Pointer movingImageReader = ReaderType::New();
   movingImageReader->SetFileName( movingImageFileName );
+  movingImageReader->Update();
 
   ShowProgress::Pointer showProgress = ShowProgress::New();
   filter->AddObserver( itk::ProgressEvent(), showProgress );
   filter->SetFixedImage( fixedImageReader->GetOutput() );
   filter->SetMovingImage( movingImageReader->GetOutput() );
 
+  //typename FilterType::ParameterObjectPointer parameterObject = filter->GetParameterObject();
```
Comment (dzenanz) [inline test/itkElastixRegistrationMethodTest.cxx]: Maybe pass an additional command line parameter, and enable this (currently commented-out code) if it is non-zero?

File: dockcross-manylinux-build-module-wheels-opencl.sh (hunk 1 above)
Comment (thewtex) [reply dockcross-manylinux-build-module-wheels-opencl.sh]: Removed

File: test/itkElastixRegistrationMethodTest.cxx (hunk 2 above)
Comment (thewtex) [reply test/itkElastixRegistrationMethodTest.cxx]: These are set in the constructor -- I'll remove these.

**Pull request merged**